content/posts/2021-01-05-understanding-non-standard-evaluation.es.Rmarkdown

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Entendiendo la Evaluación No Estándar. Parte 1: Lo Básico'
+author: Thomas Neitmann
+date: '2021-01-27'
+slug: understanding-nse-part1
```

Comment:
Could you translate the `slug` as well? This will become the URL of the post. Instead of `nse` it's probably better to use the full term.